25.md

```diff
@@ -7,20 +7,16 @@ Reactions
 
 `draft` `optional`
 
-A reaction is a `kind 7` event that is used to react to other events.
+A reaction is a `kind 7` event that is used to indicate user reactions to other events. A
+reaction's `content` field MUST include a single character indicating the value of the reaction.
+Reactions MUST NOT be used to support application-level semantics, they are for user-generated
+content only.
```

Comment:
Aren't upvotes and downvotes and tallying behind them application-level semantics? Clients should be able to rank by reactions in custom UIs if they want to. There is no need to make this limitation.

Reply:
Sure, that's not what I meant, but I've removed this line anyway.